### PR TITLE
Fix: app crash when force unwrapping conversation in share extension

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

App crash at the line `let conversation = target!` in `PostContent.send()` function.

### Causes

target conversation is nil in this case, which is cleaned in the method `updateAccount()`

### Solutions

Unwrap `target` before access.

### CODE REVIEW

File: Wire-iOS Share Extension/UnsentSendable.swift [hunk 1]
```diff
@@ -34,12 +34,17 @@ enum UnsentSendableError: Error {
     // The attachment is over file size limitation
     case fileSizeTooBig
 
+    // the target conversation does not exist anymore
+    case conversationNotExist
```
Comment (typfel) [inline Wire-iOS Share Extension/UnsentSendable.swift:38]: Why not write out the whole sentence

```suggestion
    case conversationDoesNotExist
```

File: Wire-iOS Share Extension/Base.lproj/Localizable.strings [hunk 2]
```diff
@@ -38,6 +38,7 @@
 
 "content.file.too_big" = "You can send files up to %@";
 "content.file.unsupported_attachment" = "Unsupported attachment";
+"share_extension.error.conversation_not_exist.message" = "Please select a conversation for sharing";
```
Comment (typfel) [inline Wire-iOS Share Extension/Base.lproj/Localizable.strings:41]:
```suggestion
"share_extension.error.conversation_does_not_exist.message" = "Please select a conversation for sharing";
```

Comment (typfel) [reply Wire-iOS Share Extension/Base.lproj/Localizable.strings:41]: Have this copy been approved?

Comment (billypchan) [reply Wire-iOS Share Extension/Base.lproj/Localizable.strings:41]: Not yet, I will update it.

**Pull request merged**